flow/util/autotuner/flowautotuner_hyperopt.py

```diff
@@ -115,14 +115,14 @@ def easy_objective(config):
     grOverflow = config["GR_OVERFLOW"]
 
     # parse trial config hyperparameters to genMassive.py script
-    # Newly generated genMassive.py scripts are located in ./util/autotune/runs/autotune-{variantName}.py
+    # Newly generated genMassive.py scripts are located in ./util/autotuner/runs/autotuner-{variantName}.py
```

Comment:
Make sure the path where the scripts are created are included on `.gitignore` to avoid being committed by accident.